doc/grass_database.html

```diff
@@ -86,74 +89,74 @@ <h3>GRASS Mapsets</h3>
 as well as a tool for isolating different tasks to prevent data loss.
 
 <p>
-GRASS GIS is always connected to one particular Mapset.
+GRASS GIS is always connected to one particular mapset.
```

Comment:
Isn't it rather the current session?

Reply:
We don't introduce the term session anywhere here, so I wouldn't use it. If we want to use it here, probably in a separate PR.